src/output/index.test.js

```diff
@@ -28,6 +29,32 @@ test('outputs to stderr', () => {
   expect(stdmock.flush().stderr).toEqual(['it works'])
 })
 
+describe('timestamps', () => {
+  let origFormat = moment.prototype.format
+  beforeAll(() => {
+    process.env.HEROKU_TIMESTAMPS = '1'
+    const timestamps = jest.fn()
+    timestamps.mockReturnValue('2017-06-19T07:42:57-07:00')
+    moment.prototype.format = timestamps
+  })
+
+  afterAll(() => {
+    delete process.env['HEROKU_TIMESTAMPS']
+    moment.prototype.format = origFormat
+  })
+
+  test('outputs with timestamps', () => {
+    stdmock.use()
+    const out = new Output({mock: false})
+    out.stdout.write('\n')
+    out.stdout.write('it')
+    out.stdout.write('works\n')
+    out.stdout.log('it works')
+    stdmock.restore()
+    expect(stdmock.flush().stdout).toEqual(['\n', '[2017-06-19T07:42:57-07:00] it', 'works\n', '[2017-06-19T07:42:57-07:00] it works\n'])
```

Comment:
Not when all the tests are run together, I have to ensure a newline has been sent (or I could ensure `startOfLine` is `true` manually)